[2024-06-20T04:16:17.422+0000] {scheduler_job_runner.py:769} ERROR - Executor reports task instance <TaskInstance: ELT-PROJECT.wait_until_all_connections_closed manual__2024-06-20T04:16:01.949106+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
